LibreDemoPlugin/LibreDemoCGMManager.swift

```diff
@@ -7,11 +7,12 @@
 //
 
 import Foundation
+import HealthKit
```

Comment:
[nit] was it intended to add HealthKit here?

Reply:
I just reordered it. Still being used below!